src/Extension/AssetTrait.php

```diff
@@ -180,16 +180,16 @@ private function normalizeAsset(FileAssetInterface $asset)
             return;
         }
 
-        $file = $this->getWebDirectory()->getFile($asset->getPath());
+        $app = $this->getContainer();
+
+        $file = $app['filesystem']->getFile("extensions_assets://{$this->getId()}/$path");
```

Comment:
This breaks extensions installed via Composer (CLI or API), as Composer will install into `{path}/{composer_vendor}/{composer_name}/`, so the problem is, that `getId()` will return `{path}/{BoltVendorFromNS}/{BoltNameFromClass}/` (i.e. CamelCase), which on file systems that are case-sensitive, it won't be found and is skipped.